croptalk/tools.py

```diff
@@ -289,6 +289,91 @@ def get_wfrp_commodities(reinsurance_yr: str, state_code: str, county_code: str)
         print("API call failed with status code:", response.status_code)
         return None
 
+@tool("get-SOB-recommendation")
+def get_sob_recommendation(state_code: str,
+                    commodity_code: str,
+                    insurance_plan_code: str) -> pd.DataFrame:
+    """
+    This tool's sole purpose is to find the Summary of Business (SOB) recommendation score from the recommendation system.
+    The recommendation system provide a score based on historical data of a given commodity and insurance plan and commodity.
+
+    It is meant to recommend the best possible coverage level to the user based on the historical data for a ficen state
+    , commodity and insurance plan.
+    Please provide the coverage level % that is the best and the corresponding score.
```

Comment:
the best along which criteria?

Reply:
Should we handle the case where the tool uses arguments which are not codes or strings which are not expected?